CalibCalorimetry/HcalAlgos/src/HcalTimeSlew.cc

```diff
@@ -27,7 +27,7 @@ double HcalTimeSlew::delay(double fC, ParaSource source, BiasSetting bias, doubl
   }
   else if (source==Data || source==MC){
     if(isHPD) return std::fmin(cap,tspar0[source-1]+tspar1[source-1]*log(fC+tspar2[source-1]));
-    return std::fmin(cap,tspar0_siPM[source-1]+tspar1_siPM[source-1]*log(fC+tspar2_siPM[source-1]));
+    return cap+tspar0_siPM[source-1];
```

Comment:
so, we are changing from 0 to 9, right?
Why change the meaning of cap for SiPM (min value of delay for HPD and an extra shift to t0)?